cmd/limactl/editflags/editflags.go

```diff
@@ -0,0 +1,60 @@
+package editflags
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+)
+
+// Register registers flags related to in-place YAML modification.
+func Register(cmd *cobra.Command) {
+	flags := cmd.Flags()
+
+	flags.Int("cpus", 0, "number of CPUs") // Similar to colima's --cpu
+	_ = cmd.RegisterFlagCompletionFunc("cpus", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return []string{"1", "2", "4", "8"}, cobra.ShellCompDirectiveNoFileComp
+	})
+	flags.Int("memory", 0, "memory in GiB") // colima-compatible
+	_ = cmd.RegisterFlagCompletionFunc("memory", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return []string{"1", "2", "4", "8", "16", "32"}, cobra.ShellCompDirectiveNoFileComp
+	})
```

Comment:
Should also add "disk" here, ("disk in GiB")

Reply:
I have some concern here, because disk size can only ever be increased. The only way to shrink disk size is by destroying and re-creating the instance, losing all your data.

So this is not something that you should casually do. Or it should require a confirmation prompt for the user to acknowledge (disabled by `--tty=false`).

Reply:
For new instances it can be casually done

Reply:
> For new instances it can be casually done

Yes, my concern is purely about `limectl edit default --disk 500GiB`, and then being unable to ever shrink it again.

Reply:
Could disable (or prompt) the feature to edit the disk image, until grow and shrink has been implemented ?

* https://github.com/lima-vm/lima/issues/1448

Reply:
Yes, `--disk` should not be implemented until Lima can grow the disk automatically.

FWIW, I don't expect us to ever be able to shrink disks; is this even possible?